.cargo/config_fast_builds

```diff
@@ -14,7 +14,7 @@ rustflags = ["-C", "link-arg=-fuse-ld=/usr/local/bin/zld", "-Zshare-generics=y"]
 
 [target.x86_64-pc-windows-msvc]
 linker = "rust-lld.exe"
-rustflags = ["-Zshare-generics=y"]
+rustflags = ["-Zshare-generics=n"]
```

Comment:
we should probably just remove the rustflags section entirely here. generic sharing is off by default last time i checked.

Reply:
`share-generics` *is* off by default, but only with `--release`, otherwise it's apparently on by default. This is where I got this information https://github.com/rust-lang/rust/issues/67276 and I've verified it on my machine by successfully doing `cargo +stable run --features bevy/dynamic --release`, which doesn't work without --release.